packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -100,6 +100,20 @@ class ObjCSelector {
   final String value;
 }
 
+/// Metadata to annotate methods to control the signature used for Swift output.
+///
+/// The number of components in the provided signature must match the number of
+/// arguments in the annotated method.
+/// For example:
+///   @SwiftFunction('setValue(_:for:)') double divide(int value, String key);
```

Comment:
It seems like half of this example changed at some point, and it no longer makes sense. Is the right hand side supposed to be `set(Object value, String, key)`?

Reply:
Fixed